### PR TITLE
Fix KS test p-value computation

### PR DESCRIPTION
**Pull request by MilenaTrajanoska**
## Description

Fix KS test p-value computation
Add condition in entropy calculation

### General Checklist

* [ ] Tests added for this feature/bug
      if it was a bug, test must cover it.
* [ ] Conform by the style guides, by using formatter
* [ ] Documentation updated
* [ ] (optional) Please add a label to your PR

### CODE REVIEW

File: src/whylogs/core/summaryconverters.py [hunk 1]
```diff
@@ -251,13 +253,42 @@ def ks_test_compute_p_value(target_distribution: kll_floats_sketch, reference_di
...
+
+    while j < num_quantiles:
+        cdf_target = target_distribution.get_cdf([ref_quantile_values[j]])[0]
+        cdf_ref = reference_distribution.get_cdf([ref_quantile_values[j]])[0]
+        D = abs(cdf_target - cdf_ref)
+        if D > D_max:
+            D_max = D
+        j += 1
+
     n_samples = min(target_distribution.get_n(), reference_distribution.get_n())
```
Comment (MilenaTrajanoska) [inline src/whylogs/core/summaryconverters.py:292]: Is this the right number of samples?

File: src/whylogs/core/summaryconverters.py [hunk 2]
```diff
@@ -251,13 +253,42 @@ def ks_test_compute_p_value(target_distribution: kll_floats_sketch, reference_di
...
     p_value = special.kolmogorov(np.sqrt(n_samples) * D_max)
```
Comment (MilenaTrajanoska) [inline src/whylogs/core/summaryconverters.py:293]: Is this the right distribution for the p-value?

**Comment by bernease** [comment]
Looking into this later today.

**Review by jamie256 — APPROVED** [review]
LGTM!

File: src/whylogs/core/summaryconverters.py (hunk 2 above)
Comment (jamie256) [reply src/whylogs/core/summaryconverters.py:293]: Will follow up with Bernease on these questions

**Pull request merged**